src/core/listener.h

```diff
@@ -32,15 +32,30 @@ typedef struct QUIC_LISTENER {
     BOOLEAN Stopped;
 
     //
-    // Indicates the listener was closed by the app in the stop complete event.
+    // Indicates the listener needs its stop complete event.
     //
-    BOOLEAN NeedsCleanup;
+    BOOLEAN NeedsStopCompleteEvent;
 
     //
     // Indicates the listener opted in for DoS Mode event.
     //
     BOOLEAN DosModeEventsEnabled;
 
+    //
+    // Indicates a DoS Mode event indication is required.
+    //
+    BOOLEAN NeedsDosModeModeEvent;
```

Comment:
Why are these full BOOLEANs and not part of the bitfield?

Reply:
They are unsynchronized - there is a comment where they get set, but I should add a comment here, too.